symbolic_pymc/tensorflow/printing.py

```diff
@@ -38,14 +42,58 @@ def print(self, obj):
         self.buffer.flush()
 
     def println(self, obj):
-        self.buffer.write(self.format(obj) + "\n")
-        self.buffer.flush()
+        try:
+            if None not in self.depth_index and self.depth_count in range(
+                self.depth_index[0], self.depth_index[1]
```

Comment:
Why not pass a `range`, or at least convert to one, in the constructor?

Reply:
I choose to convert to a range in my latest commit.